x-pack/platform/plugins/shared/actions/server/config.ts

```diff
@@ -121,9 +127,26 @@ export const configSchema = schema.object({
   microsoftGraphApiScope: schema.string({ defaultValue: DEFAULT_MICROSOFT_GRAPH_API_SCOPE }),
   microsoftExchangeUrl: schema.string({ defaultValue: DEFAULT_MICROSOFT_EXCHANGE_URL }),
   email: schema.maybe(
-    schema.object({
-      domain_allowlist: schema.arrayOf(schema.string()),
-    })
+    schema.object(
+      {
+        domain_allowlist: schema.maybe(schema.arrayOf(schema.string())),
+        services: schema.maybe(
+          schema.object({
+            ses: schema.object({
+              host: schema.maybe(schema.string()),
+              port: schema.maybe(schema.number()),
```

Comment:
Apologies if there is a check elsewhere in the code. Should we allow any URL here? Or can this be limited, e.g. `*.amazonaws.com`?